letsdrill.rb

```diff
@@ -1,24 +1,32 @@
 def get_letter_grade(integer)
- 
+ if integer >= 90
+   return "A"
```

Comment:
It is convention in Ruby to avoid explicit `return` statements. This should work the same without them. For instance:

```ruby
if integer >= 90
  "A"
```